optaplanner-examples/src/main/resources/org/optaplanner/examples/examination/solver/examinationSolverConfig.xml

```diff
@@ -9,6 +9,7 @@
   <entityClass>org.optaplanner.examples.examination.domain.FollowingExam</entityClass>
 
   <scoreDirectorFactory>
+<!--    <constraintProviderClass>org.optaplanner.examples.examination.optional.score.examinationConstraintProvider</constraintProviderClass>-->
     <scoreDrl>org/optaplanner/examples/examination/solver/examinationScoreRules.drl</scoreDrl>
```

Comment:
Has this been through assertion score director and `FULL_ASSERT`? Since I don't see it commented out in the example, I assume not.

Do it. It will tell you whether or not your implementation is identical to DRL.